workinggroups/README.md

```diff
@@ -0,0 +1,11 @@
+# CNCF Working Groups
+
+## Introduction
+
+The purpose of working groups are to study and report on a particular question and make recommendations based on its findings. The end result of a working group may be a new project proposal, landscape, whitepaper or even a report detailing their findings. The intention of working groups is not to host a full project or specification. Working Groups can be formed at any time but must be sponsored by a TOC member and voted with a super majority vote by the CNCF TOC. The TOC can also shut down a working group with a super majority vote.
+
+## Process
+
+If you would like to submit a working group proposal, please submit a pull request to the working groups folder. As an example, you can see the other working group proposals here: https://github.com/cncf/toc/tree/master/workinggroups
+
```

Comment:
Could it be noted that the PR's should include:

* Goals
* Non-goals
* Mailing list information
* The location of meetings / agenda / notes
* Initial interested parties to show that there are multiple people across multiple orgs interested
* The chair(s) being explicitly listed so they are discoverable

It would also be nice to note that the goals should be something that can be accomplished and that once they are complete the WG needs to revise the goals and have the revisions approved by the TOC or come to a close.